packages/core-ethereum/crates/core-ethereum-db/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "core-ethereum-db"
-version = "0.3.0"
+version = "0.4.0"
```

Comment:
I'd say that the API is backward compatible, so this is likely a patch bump.

Reply:
It changed the signature of `mark_redeemed` and `mark_losing_tickets`, thus the minor bump.